src/ubuntu/build-scripts/build-rootfs.sh

```diff
@@ -59,17 +59,17 @@ do
         docker rm -f $buildRootFSContainer
         exit 1
     fi
+done
 
-    echo "Tarring rootfs"
-    docker exec $buildRootFSContainer \
-        tar Ccf /rootfs - . >> $PWD/rootfs.tar
+echo "Tarring rootfs"
+docker exec $buildRootFSContainer \
```

Comment:
Have you tested this change?   I was expecting https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/27/files#diff-7da34d713efd60732cee8ecf5d7a78fbL25 to fail on the second iteration because a container will already exist with name $buildRootFSContainer.

Reply:
It won't fail because there are two containers with different names

But the tar will miss the first arch

Reply:
Scratch that - that container name is changing so the second iteration will pass but it won't contain the rootfs for the first arch.

Reply:
Another simpler solution is to put $arch in the rootfs.tar name.  i.e. rootfs.$arch.tar

Reply:
@sdmaclea - you're right - I realized that immediately after making the comment.

Reply:
Strangely the tar seems to be missing the second arch as arm is correctly added but x86 is not.

In addition, #16 is a nice to have; however, we need both rootfs currently.

Reply:
```
Have you tested this change? I was expecting https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/27/files#diff-7da34d713efd60732cee8ecf5d7a78fbL25 to fail on the second iteration because a container will already exist with name $buildRootFSContainer.
```

No WIP

Reply:
Having multiple arch rootfs names will lead to multiple ADD statements.

Reply:
Correct

Reply:
Taring the whole directory after the loops sounds correct to me.

Reply:
> In addition, #16 is a nice to have; however, we need both rootfs currently.

Do they really need to be in the same docker tag?

Reply:
Yes, in order to support https://github.com/dotnet/coreclr/pull/16953